Microsoft.Dynamics365.UIAutomation.Browser/Extensions/SeleniumExtensions.cs

```diff
@@ -598,6 +618,18 @@ public static IWebElement WaitUntilClickable(this ISearchContext driver, By by,
 
         #region Args / Tracing
 
+        public static Exception Throw(this IWebDriver driver, string message, Exception innerException = null)
+        {
+            driver.Quit();
+            return new InvalidOperationException(message, innerException);
+        }
+
+        public static Exception Throw<T>(this IWebDriver driver, string message, Exception innerException = null) where T : Exception
+        {
+            driver.Quit();
+            return (T) Activator.CreateInstance(typeof(T), message, innerException);
+        }
+
```

Comment:
An Exception in Login Methods do left the Browser open (strange but true), calling this methods to throw the Exception solve this problem. See my comment [hier](https://github.com/microsoft/EasyRepro/issues/752#issuecomment-584012093)